crossing-training.lic

```diff
@@ -63,9 +63,9 @@ class CrossingTraining
   def dump_junk
     return unless @settings.dump_junk
     return unless Time.now - @junk_timer > 300
-    return unless DRRoom.room_objs.size > @settings.dump_item_count
+    return unless DRRoom.room_objs.size > 13
 
-    bput('dump junk', 'You should just kick', 'You have marked this room', 'You cannot summon')
+    fput('dump junk')
```

Comment:
You had unintentional changes in here and reverted the dump junk changes from earlier

Reply:
I had an earlier version then the one that was already pushed. I was in testing. :) Should I make the changes mentioned in the

Reply:
yeah switch to the red ones and this whole hunk will go away :)

Reply:
Just did, please check. :)